p4rmyknife/modules/config.py

```diff
@@ -16,3 +16,13 @@ def install_client(args):
        p4_url = 'https://louis-test-p4k.s3.us-east-2.amazonaws.com/TestFile.sh'
       #urllib.request.urlretrieve(p4_url, 'p4v.tgz')
        urllib.request.urlretrieve(p4_url, 'TestFile.sh')
```

Comment:
You should check this call for an error as well. (try/except it) Then also check the status code for anything not in the 200s.  (Meaning the actual call was successful, but that something was wrong on the server side.)

Reply:
Then just continue your code right here.  No need for another function yet.

Reply:
That's a good idea. So if there is indeed an issue on the provider's side, we can let that be known.
I'm getting an understanding of how this works. Like you said, the code is linear. After the download of the file, it will move on to the next piece of code- in this case the check-for-file chunk that I will be writing.